src/org/wof/foodministry/FoodEntry.java

```diff
@@ -28,5 +28,29 @@ public String getDay() {
     
```

Comment:
This one is actually my bad. Can you tell me what is wrong with the method **getDay()**?